src/main/scala/Hierarchy.scala

```diff
@@ -0,0 +1,125 @@
+    def ancestors(): js.Array[this.type] = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_descendants]] */
+    def descendants(): js.Array[this.type] = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_leaves]] */
+    def leaves(): js.Array[this.type] = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#node_find]] */
+    //def find(filter: ???): ??? = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_path]] */
+    def path(target: Datum): js.Array[this.type] = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_links]] */
+    def links(): js.Array[this.type] = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#node_sum]] */
+    //def sum(value: ???): ??? = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_count]] */
+    def count(): Int = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_sort]] */
+    def sort(compare: js.Function2[this.type, this.type, Int]): this.type = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_each]] */
+    def each(function: js.Function1[this.type, Unit]): this.type = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#node_each]] */
+    //def each(function: js.Function1[this.type, Unit], that: ???): this.type = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_eachAfter]] */
+    def eachAfter(function: js.Function1[this.type, Unit]): this.type = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#node_eachAfter]] */
+    //def eachAfter(function: js.Function1[this.type, Unit], that: ???): this.type = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_eachBefore]] */
+    def eachBefore(function: js.Function1[this.type, Unit]): this.type = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#node_eachBefore]] */
+    //def eachBefore(function: js.Function1[this.type, Unit], that: ???): this.type = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#node_copy]] */
+    def copy(): this.type = js.native
+
+  }
+
+  @js.native
+  trait Packed extends js.Object { self: Hierarchy[_] =>
+    def x: js.UndefOr[Double] = js.native
+    def y: js.UndefOr[Double] = js.native
+    def r: js.UndefOr[Double] = js.native
+  }
+
+  /**
+   * @see [[pack]]
+   * @see [[https://github.com/d3/d3-hierarchy#pack]]
+   */
+  @js.native
+  trait Pack[Datum] extends js.Function1[Hierarchy[Datum], Hierarchy[Datum] with Packed] {
+
+    /** @see [[https://github.com/d3/d3-hierarchy#pack_radius]] */
+    def radius(): Double = js.native
+
+    /** @see [[https://github.com/d3/d3-hierarchy#pack_radius]] */
+    def radius(radius: js.Function1[Hierarchy[Datum], js.UndefOr[Double]]): this.type = js.native
+
+    ///** @see [[https://github.com/d3/d3-hierarchy#pack_radius]] */
+    //def radius(radius: Double): this.type = js.native
```

Comment:
Maybe include as part of an implicit syntax which accepts a `Double` and makes the constant function for you?

Reply:
Yes, good idea!